README.md

```diff
@@ -69,6 +69,18 @@ pnpm run storybook
 
 Storybook will be available at [http://localhost:26638/](http://localhost:26638/)
 
+### Stop Processes
+
+```bash
+npx dev-pm shutdown
+```
```

Comment:
It is not clear that dev-pm is used, so I would state its usage somewhere above, so this makes sense